_projects/2024/sunpy/_3dplotting.md

```diff
@@ -0,0 +1,54 @@
+---
```

Comment:
We won't be running this one this year but I wanted to keep it around for future use so I added a `_` to the name and it seems to hide it on the website.

I hope this is ok?

Reply:
I guess so it's fine. Do you maybe want to add a new directory, something along the lines of e.g. `future` instead of the year that doesn't render into the website?